src/js/components/Toolbar/__tests__/Toolbar-test.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import 'jest-styled-components';
+
+import { Data } from '../../Data';
+import { DataFilters } from '../../DataFilters';
+import { DataSearch } from '../../DataSearch';
+import { Grommet } from '../../Grommet';
+import { Toolbar } from '..';
+
+const data = [{ name: 'a' }, { name: 'b' }];
+
+describe('DataFilters', () => {
+  test('renders', () => {
+    const { container } = render(
+      <Grommet>
+        <Data data={data}>
```

Comment:
looks strange here like there is an extra `gap` 

<img width="560" alt="Screen Shot 2022-12-16 at 1 25 31 PM" src="https://user-images.githubusercontent.com/42451602/208183282-0d53b10a-e9bc-4d91-929c-a77c8598a60d.png">

Reply:
but above you have `gap='small'` soo not sure what it could be

Reply:
Refactored Toolbar and the story.

Reply:
sounds good I can take a look